spec/unit/resource/windows_feature_powershell_spec.rb

```diff
@@ -35,10 +35,6 @@
     expect(resource.feature_name).to eql(%w{snmp dhcp})
   end
 
-  it "sets the default action as :install" do
-    expect(resource.action).to eql([:install])
-  end
```

Comment:
this shouldn't change?

Reply:
@lamont This test was redundant so removed one of them https://github.com/chef/chef/blob/master/spec/unit/resource/windows_feature_powershell_spec.rb#L30:L38